From ea6527dba9c8d57434a3eef8bb1793bfff8f5abb Mon Sep 17 00:00:00 2001
From: Jung Zhao <jung.zhao@rock-chips.com>
Date: Fri, 9 Dec 2016 21:11:25 +0800
Subject: [PATCH] arm64: dts: rk3399-android-next: add iommu_enabled property
 in vpu&rkvdec node

camera will query this property, this is historical issue!!!

Change-Id: Ida221e9f5781dcc5562cad69e2ac437290b946bb
Signed-off-by: Jung Zhao <jung.zhao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
index dab45c1d53d5..37aa5b352255 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
@@ -139,6 +139,7 @@
 		compatible = "rockchip,vpu_service";
 		rockchip,grf = <&grf>;
 		iommus = <&vpu_mmu>;
+		iommu_enabled = <1>;
 		reg = <0x0 0xff650000 0x0 0x800>;
 		interrupts = <GIC_SPI 113 IRQ_TYPE_LEVEL_HIGH 0>,
 			<GIC_SPI 114 IRQ_TYPE_LEVEL_HIGH 0>;
@@ -168,6 +169,7 @@
 		compatible = "rockchip,rkvdec";
 		rockchip,grf = <&grf>;
 		iommus = <&vdec_mmu>;
+		iommu_enabled = <1>;
 		reg = <0x0 0xff660000 0x0 0x400>;
 		interrupts = <GIC_SPI 116 IRQ_TYPE_LEVEL_HIGH 0>;
 		interrupt-names = "irq_dec";
-- 
2.35.3

